recipe/meta.yaml

```diff
@@ -1,39 +1,71 @@
 {% set name = "sagemaker_inference" %}
-{% set version = "1.5.5" %}
+{% set version = "1.5.6" %}
+
 
 package:
-  name: sagemaker-inference-toolkit
+  name: {{ name|lower }}-meta
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 20f878c8608768f6a8ed850417af31f4790e400bc1fdb1742c87d736a33c119b
+  sha256: cb193713448e4ad26cc3e5652251880b086f2814800b2310c702cb257ce5ec13
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  host:
-    - pip
-    - python
-  run:
-    - numpy
-    - python
-    - six
-    - typing
-
-test:
-  imports:
-    - sagemaker_inference
+  number: 0
+
+outputs:
+  - name: sagemaker-inference
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
      script: python -m pip install . -vv
```

Reply:
Any idea why that's not working? Should theoretically be correct, right?

Reply:
@mariusvniekerk It looks like it's now pulling in the deps as well since the full python command is not passed. 
What would be the string that would get substituted for `{{ PYTHON }}` so that I can manually add it?

And do you know if this is fixable for the future?

Reply:
So you cannot use script like this for multioutput. You must have an actual script file to execute afaik.

Reply:
That was it!